Makefile

```
@@ -64,6 +64,10 @@ URL_IGNORE=\#$\
           ,/.*web.libera.chat.*/$\
           ,/^http://tracing.istio-system.*/$\
           ,/^https://tracing-service.*/
+
+NEW_URLS=$(shell scripts/ignore_new_urls.sh)
+URL_IGNORE:=$(URL_IGNORE)$(NEW_URLS)
```

I am not familiar with this kind of syntax in a makefile. I think you just want URL_IGNORE to be the stdout that is printed by the script right? This seems to be what you want:

```suggestion
URL_IGNORE ?= $(shell scripts/ignore_new_urls.sh 2> /dev/null)
```

Note the `?=` means that the caller of the makefile can override this if they want via the make cmdline option ` -e` (useful for testing the make target, for example).

We also want that `2> /dev/null` because we want to redirect the stderr of the script to dev/null because you don't want error messages considered URLs which might cause errors later in the make invocation.

---

Agree with `2> /dev/null`

But, in the case of 
`URL_IGNORE ?= $(shell scripts/ignore_new_urls.sh 2> /dev/null)`

I would like to concat the existent defined urls with the ones that the script returns - Should be dynamic to just avoid these 404. The ones already defined in the makefile as URL_IGNORED they always should be ignored - I think, I didn't want to change this behavior.

---

> I would like to concat the existent defined urls with the ones that the script returns

Oh, duh! I didn't notice URL_IGNORE was defined above. I get it now :)